### ISSUE TITLE
KeyError: 'browser'

### ISSUE THREAD

**Issue by mgeorgetw** [thp/urlwatch#231]
I was trying the Browser feature with this YAML:

```yaml
kind: browser
name: Google
navigate: https://google.com/
```

but all I could get was KeyError:

```bash
Traceback (most recent call last):
  File "/usr/local/bin/urlwatch", line 107, in <module>
    urlwatch = Urlwatch(command_config, config_storage, cache_storage, urls_storage)
  File "/usr/local/lib/python3.6/site-packages/urlwatch/main.py", line 63, in __init__
    self.load_jobs()
  File "/usr/local/lib/python3.6/site-packages/urlwatch/main.py", line 84, in load_jobs
    jobs = self.urls_storage.load_secure()
  File "/usr/local/lib/python3.6/site-packages/urlwatch/storage.py", line 244, in load_secure
    jobs = self.load()
  File "/usr/local/lib/python3.6/site-packages/urlwatch/storage.py", line 333, in load
    return [JobBase.unserialize(job) for job in yaml.load_all(fp) if job is not None]
  File "/usr/local/lib/python3.6/site-packages/urlwatch/storage.py", line 333, in <listcomp>
    return [JobBase.unserialize(job) for job in yaml.load_all(fp) if job is not None]
  File "/usr/local/lib/python3.6/site-packages/urlwatch/jobs.py", line 124, in unserialize
    return cls.__subclasses__[kind].from_dict(data)
KeyError: 'browser'
```

I have installed `requests-html` and had been using the old `kind: url` for a while without a problem. So I am wondering if I missed something obvious. Thanks.

**Comment by thp** [comment]
The browser support hasn't been released yet, you have to get urlwatch from Git to have that feature.

**Comment by mgeorgetw** [comment]
Didn't notice that. Thanks!

**Comment by thp** [comment]
This shipped with urlwatch 2.10. Closing.
